libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -128,6 +128,13 @@ float AP_RSSI::read_receiver_rssi()
         case RssiType::RSSI_RC_CHANNEL_VALUE :
             receiver_rssi = read_channel_rssi();
             break;
+        case RssiType::RSSI_RECEIVER : {
+            int16_t rssi = hal.rcin->get_rssi();
+            if (rssi != -1) {
+                receiver_rssi = rssi / 255.0;
+            }
```

Comment:
In my original PR, I included SBUS_LOW and SBUS_HIGH parameters, to function like the CHAN_LOW/CHAN_HIGH and PIN_LOW/PIN_HIGH values. Shouldn't we also have a RECEIVER_PROTOCOL_LOW/RECEIVER_PROTOCOL_HIGH?

I think it's important that the user get to define exactly what the RSSI range is **for them**. This allows us to be flexible and not assume anything about a users use case.